network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_blocks.html

```diff
@@ -35,6 +35,10 @@
 
             <div class="full-profile">
               <a href="https://www.mozillapulse.org/profile/{{ profile.profile_id }}">SEE FULL PROFILE →</a>
+              <div className="person-social-links mt-3">
+                <a href="" class="twitter gray small mr-4"></a>
```

Comment:
Probably worth "conditioning" this: `{% if profile.twitter %}<a href="{{ profile.twitter }}"> class="...">...</a>{% endif %}`

Reply:
Good point.

Reply:
In this case, it's profile.twitter, but for the actual Fellows page when we make the updates it would be fellow.twitter, right?

Reply:
it'll still be `profile`, since we'll simply iterating through the profiles using `{% for profile in profiles %} .... {% endfor %}`.